internal/dag/listener_processor.go

```diff
@@ -15,6 +15,12 @@ package dag
 
 import "sort"
 
+// nolint:revive
+const (
+	HTTP_LISTENER_NAME  = "ingress_http"
+	HTTPS_LISTENER_NAME = "ingress_https"
+)
```

Comment:
I wasn't quite sure if it made sense to have these + the constants currently defined in internal/xdscache/v3 be consolidated (meaning the DAG listener names always equal the Envoy listener names), so for now I left them separate. Open to input here.

Reply:
I think this is fine for now. As @stevesloka said in another comment, if we do multiple listeners, all of this is going to need to change anyway.